linux/system/kernel.sls

```diff
@@ -53,6 +53,21 @@ linux_kernel_module_{{ module }}:
 
 {%- endfor %}
 
+{%- for module_name, module_content in system.kernel.get('module', {}).iteritems() %}
```

Comment:
there maybe such case that you need to blacklist a module, and to do that you need to have this file present with `blacklist {{module_name}}` content, this can be misleading, because we have variable `enabled: true` and at the same variable `blacklist: true`, which is contradictory in my opinion.